dts/bindings/usb/raspberrypi,pico-usbd.yaml

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2021, Pete Johanson
+# SPDX-License-Identifier: Apache-2.0
+
+description: RaspberryPi Pico USB Device Controller
+
+compatible: "raspberrypi,pico-usbd"
+
+include: usb-ep.yaml
```

Comment:
Please also make USB a reset device. See #45131 for an example

Reply:
Done.